packages/ui-pattern-block/src/hooks/useDebounce.tsx

```diff
@@ -0,0 +1,14 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Can't we use the `useDebounce` of `fondue`?

Reply:
I tried it but that was not working properly when I used it in a function in a component.